apps/src/Sounds.js

```diff
@@ -259,6 +259,16 @@ Sounds.prototype.play = function (soundId, options) {
   if (sound) {
     sound.play(options);
   }
+  this.audioStartedTime = this.audioContext.currentTime;
+};
+
+//Returns time since the current audio started to play
+Sounds.prototype.getCurrentTime = function () {
+  if (this.audioStartedTime) {
+    return this.audioContext.currentTime - this.audioStartedTime;
+  } else {
+    return 0;
+  }
```

Comment:
For the HTML5 audio case, you'll probably be able to use `audioElement.currentTime`, but I'm not sure how accurate that will be. We may be better off capturing `this.audioStartedTime` for both type of audio playback using something like this: https://developer.mozilla.org/en-US/docs/Web/API/Performance/now

Reply:
Also, this seems odd to place on the `Sounds` object, as we allow multiple sounds to play simultaneously.